pypsa/optimization/global_constraints.py

```diff
@@ -389,12 +391,8 @@ def define_transmission_volume_expansion_limit(n, sns):
     m = n.model
     glcs = n.global_constraints.query("type == 'transmission_volume_expansion_limit'")
 
-    def substr(s):
-        return re.sub("[\\[\\]\\(\\)]", "", s)
-
-    lhs = []
     for name, glc in glcs.iterrows():
-        car = [substr(c.strip()) for c in glc.carrier_attribute.split(",")]
```

Comment:
I think `car` is still needed. It's used a few lines down in a query string.

Reply:
Ah, yes, damn. Never trust VS Code blindly. The inquery string usage was lost on it.